tools/who_what_benchmark/tests/test_cli_image.py

```diff
@@ -55,6 +55,7 @@ def get_similarity(output: str) -> float:
     return float(matches[-1])
 
 
+@pytest.mark.common_scope
```

Comment:
I'm a bit confused with `common_scope` mark. How we define what is common what's not?

Reply:
> Maybe rename it to `@pytest.mark.default_transformers` to make it less confusing?

@sbalandi , @popovaan don't you mind to revert this `common_scope` mark?
`not nanollava` seems more simple, doesn't require to add `@pytest.mark.default_transformers` for each new test.
What do you think?

Reply:
I'm ok with `not nanollava` as condition, @popovaan sorry for inconvenience